test/config/utility.cc

```diff
@@ -1211,6 +1212,49 @@ bool ConfigHelper::setListenerAccessLog(const std::string& filename, absl::strin
   return true;
 }
 
+void ConfigHelper::initializeTlsKeyLog(
+    envoy::extensions::transport_sockets::tls::v3::CommonTlsContext& tls_context,
+    bool keylog_local_filter, bool keylog_remote_filter, bool keylog_negative,
+    const Network::Address::IpVersion version) {
+  if (!keylog_local_filter && !keylog_local_filter) {
+    return;
+  }
+  auto tls_keylog_path = tls_context.mutable_tls_keylog()->mutable_logfile_path();
+  if (!keylog_negative) {
+    *tls_keylog_path = TestEnvironment::temporaryPath("tlskey.log");
+  } else {
+    *tls_keylog_path = TestEnvironment::temporaryPath("tlskey-negative.log");
```

Comment:
Why use different paths for the two cases?

Reply:
There are several test cases. In the first several test cases, the tlskey.log has some data already.  Another way is to clear the data in tlskey.log for every test case.